maven-plugins/helidon-archetype-maven-plugin/src/it/projects/test1/archetype3/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.build-tools.archetype.tests</groupId>
+        <artifactId>parent</artifactId>
+        <version>@project.version@</version>
+    </parent>
+    <artifactId>test-archetype3</artifactId>
+    <name>Test Archetype 3</name>
+    <packaging>helidon-archetype</packaging>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.helidon.build-tools</groupId>
+                <artifactId>helidon-archetype-maven-plugin</artifactId>
+                <version>${project.version}</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <dependency>
```

Comment:
I forgot did we agree that we'd have the schema GAV configurable ? We release both of them under the same versions, so the GAV can be fully hard-coded.

Reply:
Yes I do remember that. However here you are not declaring a dependency, you are declaring plugin configuration that describes a dependency.

You need add the dependency to the plugin pom.xml. If you cannot use the class-loader to get the xsd file as a resource then you will need to use aether to access the artifact from the maven repository.